content/en/user-guide/aws/kinesis/index.md

```diff
@@ -1,57 +1,194 @@
 ---
 title: "Kinesis"
-categories: ["LocalStack Community"]
+linkTitle: "Kinesis"
 description: >
-  Get started with AWS Kinesis on LocalStack
-aliases:
-  - /aws/kinesis/
+  Get started with Kinesis on LocalStack
 ---
 
-AWS Kinesis is a data streaming service, which enables your application to ingest, buffer, and process data in real-time. 
-Kinesis in LocalStack Community is powered by [Kinesis Mock](https://github.com/etspaceman/kinesis-mock) and has an [extensively supported API]({{< ref "feature-coverage.md" >}}).
+## Introduction
 
-## Quickstart
+Kinesis is a platform provided by Amazon Web Services (AWS) that enables your application to ingest, buffer, and process data in real-time. Kinesis is suitable for applications that require processing and deriving insights from data streams such as logs, metrics, user interactions, and IoT sensor readings. Kinesis offers three main services: Kinesis Data Streams, Kinesis Data Firehose, and Kinesis Data Analytics. In this page, we take a look at Kinesis Data Streams which allows you to capture and store real-time data streams.
```

Comment:
It might be a bit confusing to list "Kinesis Data Streams, Kinesis Data Firehose, and Kinesis Data Analytics" but not having "Kinesis Data Streams" in our list of services?

Reply:
This is for in-general AWS understanding. The LocalStack coverage is discussed in the second paragraph.